spec/habitat_spec.cr

```diff
@@ -10,6 +13,7 @@ class FakeServer
     setting something_that_can_be_multiple_types : String | Int32
     setting this_can_be_nil : String?
     setting nilable_with_default : String? = "default"
+    setting constant_setting : RandomClass.class | Nil, example: "RandomClass"
```

Comment:
```suggestion
    setting constant_setting : RandomClass.class
```

I believe the test case is a non-nilable class with no example text. When the setting is not set it should raise the helpful error message with `Try this... RandomClass.class`

Reply:
Ah, ok. I'll try that. Thanks!

Reply:
oh, looks like if you don't set `example`, then it'll always default to `some_value` https://github.com/luckyframework/habitat/blob/74db60a817ce326545d420b038f7da9758383727/src/habitat.cr#L7

```
     The 'constant_setting' setting for FakeServer was nil, but the setting is required.

       Try this...

         FakeServer.configure do |settings|
           settings.constant_setting = some_value
         end
```

But if I do set the value, then the specs all pass. I guess we could also pass in the setting's type for this error so whether it's a `String`, or a `RandomClass.class`, it'll always show that 🤔

Reply:
I wonder if that issue was already fixed, and just never got closed. [this commit](https://github.com/luckyframework/habitat/commit/26133439ad2fd248321e715f7c6abde85b826354) was done on Feb 16th 2019 which looks like it may have caused that issue that was reported on Feb 22nd 2019...